apps/web/src/views/Farms/components/FarmTable/FarmTable.tsx

```diff
@@ -196,6 +226,15 @@ const FarmTable: React.FC<React.PropsWithChildren<ITableProps>> = ({ farms, cake
         return row
       }
 
+      // V3
+      const farmCakePerSecond =
+        farm.allocPoint && totalAllocPoint && latestPeriodCakePerSecond
+          ? ((ethersToBigNumber(farm.allocPoint).toNumber() / ethersToBigNumber(totalAllocPoint).toNumber()) *
+              ethersToBigNumber(latestPeriodCakePerSecond).toNumber()) /
+            PRECISION.toNumber() /
+            1e18
+          : 0
```

Comment:
you should already `cakePerSecond` from useFarmsV3

Reply:
The `cakePerSecond` of `useFarmsV3` only includes the sum of all V3 farms. Here, it is necessary to calculate the `cakePerSecond` for each individual farm.

Reply:
poolWeight * cakePerSecond

Reply:
fixed